client/components/App.jsx

```diff
@@ -1,19 +1,17 @@
 import React from 'react';
-import { render } from 'react-dom';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
```

Comment:
Since we moved Router to render.jsx, I think this BrowserRouter as Router here is redundant and can be removed.